### PR TITLE
feat: 5.2.0 dev stage UI

### PR DESCRIPTION
**Pull request by MillenniumFalconMechanic**
## Reason for Change

- #6994 

## Changes

- Updated development stage constants.

image

## Testing steps

- Manually test.

## Checklist 🛎️

- [ ] Add product, design, and eng as reviewers for rdev review

### CODE REVIEW

File: frontend/tests/features/filter/constants.ts [hunk 1]
```diff
@@ -0,0 +1,240 @@
+import {
```
Comment (MillenniumFalconMechanic) [inline frontend/tests/features/filter/constants.ts:1]: Added this "pinned" test version of ontology term set as tests should be covering functionality not data; this prevents any future updates to schema from affecting tests.

File: frontend/tests/features/filter/useCategoryFilter.test.ts
Comment (MillenniumFalconMechanic) [inline frontend/tests/features/filter/useCategoryFilter.test.ts:1]: Updates in this file are specific to using [test ontology term set](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7345/files#diff-035689bc05b1f5925322a2922a7af204d3823c516d032c66f8ef3eeb7be0e34cR1) and not [app schema-driven term set](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7345/files#diff-aedcb2b9969695dce155a593352e233caf040ea1bf099848e09f08d059e30edbR26).

File: frontend/tests/features/filter/utils.test.ts
Comment (MillenniumFalconMechanic) [inline frontend/tests/features/filter/utils.test.ts:1]: Updates in this file are specific to using [test ontology term set](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7345/files#diff-035689bc05b1f5925322a2922a7af204d3823c516d032c66f8ef3eeb7be0e34cR1) and not [app schema-driven term set](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7345/files#diff-aedcb2b9969695dce155a593352e233caf040ea1bf099848e09f08d059e30edbR26).

File: frontend/tests/features/filter/constants.ts
Comment (MillenniumFalconMechanic) [inline frontend/tests/features/filter/constants.ts:1]: Added this "pinned" test version of ontology term set as tests should be covering functionality not data; this prevents any future updates to schema from affecting tests.

**Review by nayib-jose-gloria — APPROVED** [review]

**Pull request merged**